src/p2pool_api.h

```diff
@@ -25,13 +25,14 @@ namespace p2pool {
 class p2pool_api
 {
 public:
-	explicit p2pool_api(const std::string& api_path);
+	explicit p2pool_api(const std::string& api_path, const bool local_stats);
```

Comment:
No need for `explicit` keyword anymore since constructor has 2 parameters now.

Reply:
Removed, push pending.